tests/react-a11y-mouse-event-has-key-event/test.tsx.lint

```diff
@@ -7,9 +7,11 @@ const elementTwo = (<div onMouseOut={() => {}}>click</div>)
 const elementSelfClosing = (<div onMouseOut={() => {}}/>)
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~  [onMouseOut must be accompanied by onBlur.]
 
-<div onMouseOver={() => {}} {...props} />
-<div onMouseOut={() => {}} {...props} />
-<div />
-<div {...props} />
+const Foo = (<div onMouseOver={() => {}} {...props} />)
+            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [onMouseOver must be accompanied by onFocus.]
+const Bar = (<div onMouseOut={() => {}} {...props} />)
+            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [onMouseOut must be accompanied by onBlur.]
```

Comment:
Complains for now.

Reply:
@lizzzp1 could you change it to not complain if there's a `...`, actually? It would be annoying to have a `...props` that includes the correct corresponding `onBlue`/`onFocus` and still have this complain.

Reply:
I thought that required the type checker that was a follow up? I'm really confused now.

Reply:
The type checker is just required to know whether the `...props` contains the corresponding `onBlur`/`onFocus`. Without the type checker, we have know way of knowing whether a JSX element that contains `...props` violates the rule or not. Until the type checker logic is added, that leaves us with two options:

* Ignore `...props` _(pretend they aren't therE): the rule would give complaints in some places where it shouldn't
* Assume `...props` contains the right attributes: the rule wouldn't give complaints in some places where it should

In order to not irritate folks with false complaints, IMO we should err on the side of caution with the first option.

Does that seem reasonable?

Reply:
Ok, if props are ignored then it would complain like the above wouldn't it? Wouldn't the 2nd option be better so it doesn't complain if it encounters spread props?

Reply:
Oh agh sorry I _meant_ to say the second option! You're absolutely correct 😅